README.md

```diff
@@ -141,8 +148,6 @@ And I also want to thank all my brothers for their advice.
 
 ## 📝 License <a name="license"></a>
 
-This project is [LICENCE.md](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] As I mentioned in the previous review, I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only rename/edit this `[MIT](./LICENSE)` to `[MIT](./MIT.md)` as your License file is named `MIT.md` and not `LICENSE`